server/views/pages/contacts/manage/contactDetails/details/index.njk

```diff
@@ -44,7 +44,12 @@
           </li>
           <li class="govuk-tabs__list-item">
             <a class="govuk-tabs__tab restrictions-tab-title" href="#restrictions">
-              Restrictions ({{ contactGlobalRestrictions.length + prisonerContactRestrictions.length }})
+              {% if FEATURE_SHOW_PRISONER_RESTRICTIONS_ENABLED %}
+                Restrictions ({{ contactGlobalRestrictions.length + prisonerContactRestrictions.length + prisonerRestrictions.length}})
+              {% endif %}
+                {% if not FEATURE_SHOW_PRISONER_RESTRICTIONS_ENABLED %}
+                    Restrictions ({{ contactGlobalRestrictions.length + prisonerContactRestrictions.length }})
+                {% endif %}
             </a>
```

Comment:
Isn't this just a simple `if ... then .... elif ...endif` construct? Doesn't need two if statements.